src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -585,6 +592,15 @@ public Builder appCommandOutputFormat(String appCommandOutputFormat) {
     }
 
     /**
+     * Sets structured json logs for the stderr output. Supported values include 'never' (default),
+     * 'always', 'terminal', etc.
+     */
+    public Builder appCommandShowStructuredLogs(String appCommandShowStructuredLogs) {
```

Comment:
I agree the name isn't the best though; `useStructuredLogs` doesn't help much though because if you look at the values it accepts, the name should be something like `setStructuredLogsScope` or something like that